apps/spruce/src/components/Icon/icons/FallTree.tsx

```diff
@@ -0,0 +1,127 @@
+import { useEffect } from "react";
+
+export const FallTree: React.FC = () => {
+  // Animations should be paused by default on page load.
+  useEffect(() => {
+    const fallTree = document.getElementsByTagName("svg");
+    fallTree[0].pauseAnimations();
```

Comment:
If we keep this approach can we target the svg using a ref or an id? This would match all svgs on the page.